jsonrpc/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@sourcegraph/vscode-jsonrpc",
 	"description": "A json rpc implementation over streams",
-	"version": "3.3.0-alpha.1",
+	"version": "3.3.0-alpha.1.sourcegraph.2",
```

Comment:
We are creating a patch on top of some base upstream version. This version scheme makes it clear what the upstream version our patch is based on.

It was a bit sloppy to just push `3.3.0-alpha.1` instead of something like `3.3.0-alpha.1.sourcegraph.1`